src/main/resources/public/jsx/Panel.jsx

```diff
@@ -91,20 +91,24 @@ class Panel extends Component {
                 </div>
                 <ul className="list-group">
                     <li className="list-group-item">
-                        <label className="col-sm-3 control-label">{ t('dashboard.panel.dataset_name') }</label>{log.dcModelMapping.name}
+                        <label className="col-sm-4 control-label">{ t('dashboard.panel.dataset_name') }</label>{log.dcModelMapping.name}
                     </li>
                     <li className="list-group-item">
-                        <label className="col-sm-3 control-label">{ t('dashboard.panel.dcmodel_name') }</label>{log.dcModelMapping.type}
+                        <label className="col-sm-4 control-label">{ t('dashboard.panel.dcmodel_name') }</label>{log.dcModelMapping.type}
                     </li>
+                    {renderIf(log.dcModelMapping.organization > '') (
```

Comment:
It works but not really natural for testing something is not empty (even more when you said in DcModelMapping that it should be not null and not empty) :)